tests/unit-tests/syndication.php

```diff
@@ -5,6 +5,7 @@
  * @package Newspack\Tests
```

Comment:
Nit: this test file is now more about `Optional_Modules`, so the file name could be updated.